crates/chia-consensus/src/gen/conditions.rs

```diff
@@ -99,14 +119,14 @@ impl SpendVisitor for MempoolVisitor {
             Condition::AggSigMe(_, _)
             | Condition::AggSigParent(_, _)
             | Condition::AggSigParentAmount(_, _)
-            | Condition::AggSigParentPuzzle(_, _) => {
+            | Condition::AggSigParentPuzzle(_, _)
```

Comment:
Can you think of why we restrict committing to the parent puzzle? it's supposed to be unchanged anyway, right?

Reply:
That's our Parent ID and our Puzzle, not our parent's puzzle. Ambiguous name.